sigs.yaml

```diff
@@ -3575,6 +3575,57 @@ workinggroups:
     liaison:
       github: pohly
       name: Patrick Ohly
+- dir: wg-ai-gateway
+  name: AI Gateway
+  mission_statement: >
+    The AI Gateway Working Group focuses on the intersection of AI and networking,
+    particularly in the context of extending load-balancer, gateway and proxy technologies
+    to manage and route traffic for AI Inference.
+
+  charter_link: charter.md
+  stakeholder_sigs:
+  - Network
+  label: ai-gateway
+  leadership:
+    chairs:
+    - github: keithmattix
+      name: Keith Mattix
+      company: Microsoft
+      email: keithmattix2@gmail.com
+    - github: kflynn
+      name: Flynn
+      company: Buoyant
+      email: flynn@buoyant.io
+    - github: kfswain
+      name: Kellen Swain
+      company: Google
+      email: kfswain@google.com
+    - github: nirrozenbaum
+      name: Nir Rozenbaum
+      company: IBM
+      email: nirro@il.ibm.com
+    - github: shaneutt
+      name: Shane Utt
+      company: Red Hat
+      email: sutt@redhat.com
+    - github: xunzhuo
+      name: Xunzhuo
+      company: Tencent
+      email: mixdeers@gmail.com
```

Comment:
I've raised a similar concern when reviewing WG Node Lifecycle, are you sure you want to have that many leads? I know from my personal experience that having too many makes it sometimes challenging. Definitely not a blocker for WG creation, more like a suggestion :wink:

Reply:
Indeed, our list is big, but that is reflective of a large wave of interest. Each person on this list is representative of some technical aspect of the subject matter which makes them specialists/experts for the group. Each of them has spoken with me personally and I'm confident in their commitment to dedicate substantial time to the project, including attending and leading meetings, as well as actively driving and contributing to proposals.